Svc/CmdSplitter/test/ut/CmdSplitterTester.cpp

```diff
@@ -116,7 +116,7 @@ void CmdSplitterTester ::test_response_forwarding() {
     Fw::CmdResponse response;
     response.e = static_cast<Fw::CmdResponse::T>(STest::Pick::lowerUpper(0, Fw::CmdResponse::NUM_CONSTANTS));
     U32 cmdSeq = static_cast<U32>(STest::Pick::any());
-    this->active_command_source = static_cast<NATIVE_INT_TYPE>(STest::Pick::lowerUpper(
```

Comment:
Seems an unrelated change.  Is this recommended instead of lowerUpper?

Reply:
Yes, this fixes an off-by-one error that caused an intermittent out-of-bounds array access. If your array has 10 elements and you call `lowerUpper(0,10)` you can get an index of 10, which is out of bounds. If you call `startLength(0, 10)` you will get an element in the range [0,9].